src/EditorFeatures/TestUtilities/Diagnostics/AbstractDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -236,43 +236,43 @@ private void AssertNoAnalyzerExceptionDiagnostics(IEnumerable<Diagnostic> diagno
         private static readonly CodeStyleOption<ParenthesesPreference> RemoveIfUnnecessaryPreference =
             new CodeStyleOption<ParenthesesPreference>(ParenthesesPreference.NeverIfUnnecessary, NotificationOption.Suggestion);
 
-        private static IEnumerable<PerLanguageOption<CodeStyleOption<ParenthesesPreference>>> GetAllExceptOtherParenthesesOptions()
+        internal static IEnumerable<PerLanguageOption<CodeStyleOption<ParenthesesPreference>>> GetAllExceptOtherParenthesesOptions()
         {
             yield return CodeStyleOptions.ArithmeticBinaryParentheses;
             yield return CodeStyleOptions.RelationalBinaryParentheses;
             yield return CodeStyleOptions.OtherBinaryParentheses;
         }
 
-        protected IDictionary<OptionKey, object> RequireArithmeticBinaryParenthesesForClarity
-            => GetSingleRequireOption(CodeStyleOptions.ArithmeticBinaryParentheses);
+        public static IDictionary<OptionKey, object> RequireArithmeticBinaryParenthesesForClarity(string language)
```

Comment:
Should these static properties just be moved to separate static helper class?

Reply:
Maybe. Seems safe to leave it for a future change.